workflow/rules/trim_3prime.smk

```diff
@@ -1,4 +1,4 @@
-if is_3prime_experiment:
+if is_3prime_experiment and three_prime_vendor:
```

Comment:
Must be three_prime_vendor == "lexogen":

Reply:
Noted. Added